controllers/storagecluster/storageclient.go

```diff
@@ -42,9 +41,14 @@ func (s *storageClient) ensureCreated(r *StorageClusterReconciler, storagecluste
 
 	storageClient := &ocsclientv1a1.StorageClient{}
 	storageClient.Name = storagecluster.Name
+	storageClusterNamespace := storagecluster.Namespace
```

Comment:
Why are we assuming storage cluster namespace and not operator namespace or pod namespace?

Reply:
https://github.com/red-hat-storage/ocs-operator/pull/2827#discussion_r1782099611

Reply:
changed to operator namespace , ack